src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    public enum Color {
+        RED,
+        BLUE,
+        GREEN,
+        ORANGE;
+    }
+
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        Random random = new Random();
```

Comment:
Make random class lever field. 

It should be instantiated once and reused.
```suggestion
        Random random = new Random();
```